### PR TITLE
✨Source Zendesk Support: New Stream - User Fields

### LINKED ISSUE #19123: Source Zendesk-Support - Add new stream user_fields
## Summary
- [ ] Support new stream user_fields 

## TODOs

- [Issue] Allow users to extract data also from user_fields stream using the Zendesk-support connector

## doc Zendesk
https://developer.zendesk.com/api-reference/ticketing/users/user_fields/

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
Closes #19123

## Enhancement
New streams: 

- User Fields

## Recommended reading order
1. `source_zendesk_support/source.py`
2. `source_zendesk_support/streams.py`
3.  `source_zendesk_support/schemas/*`

## 🚨 User Impact 🚨
No breaking changes

## Pre-merge Checklist
<details open><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter [Yes]

- [x] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [x] Issue acceptance criteria met
- [x] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)
- [x] Documentation which references the generator is updated as needed

### CODE REVIEW

**Comment by bmcgilli** [comment]
@lazebnyi can you please assign for review?

File: docs/integrations/sources/zendesk-support.md [hunk 1]
```diff
@@ -65,6 +65,7 @@ The Zendesk Support source connector supports the following streams:
 - [Ticket Metric Events](https://developer.zendesk.com/api-reference/ticketing/tickets/ticket_metric_events/) \(Incremental\)
 - [Ticket Skips](https://developer.zendesk.com/api-reference/ticketing/tickets/ticket_skips/) \(Incremental\)
 - [Users](https://developer.zendesk.com/api-reference/ticketing/ticket-management/incremental_exports/#incremental-user-export) \(Incremental\)
+- [UserFields](https://developer.zendesk.com/api-reference/ticketing/users/user_fields/#list-user-fields) \(Incremental\)
```
Comment (artem1205) [inline docs/integrations/sources/zendesk-support.md:68]: Incremental support is mentioned here, but I can see only **full refresh** support from code. Which statement is correct?

**Review by artem1205 — CHANGES_REQUESTED** [review]

**Comment by artem1205** [comment]
@btkcodedev , please check the comment above

**Comment by btkcodedev** [comment]
@artem1205 Changes done :)

**Comment by artem1205** [comment]
@btkcodedev , if the stream is not `incremental`, please remove redundant mentions from `abnormal_state.json` and `incremental_catalog.json`

**Comment by btkcodedev** [comment]
@artem1205 Removed unwanted :)

**Comment by artem1205** [comment]
Merged in https://github.com/airbytehq/airbyte/pull/29891

**Pull request closed without merging**